src/js/components/tour/guided.tsx

```diff
@@ -0,0 +1,64 @@
+import i18n from 'i18next';
+import React, { useCallback, useState } from 'react';
+import Joyride, {
+  ACTIONS,
+  CallBackProps,
+  STATUS,
+  status as StatusType,
+  Step,
+  StoreHelpers,
+} from 'react-joyride';
+
+import tourStyles from '~js/components/tour/styles';
+
+interface Props {
+  steps: Step[];
+  run: boolean;
+  onClose: () => void;
+}
+
+const GuidedTour = ({ steps, run, onClose }: Props) => {
+  const [helpers, setHelpers] = useState<StoreHelpers | null>(null);
+
+  const handleCallback = useCallback(
```

Comment:
@dvdherron We basically use `useCallback` any time a function is passed as an argument to another React component. At a high level, it basically ensures that the function does not change unless one of its dependencies changes -- that way we're not triggering unnecessary re-renders of the child component.

https://reactjs.org/docs/hooks-reference.html#usecallback